src/mono/netcore/sample/wasm/console/WasmSample.csproj

```diff
@@ -53,6 +54,19 @@
     <Compile Include="Program.cs" />
   </ItemGroup>
 
+  <Target Name="PrepareRunScript" BeforeTargets="CopySampleAppToHelixTestDir" Condition="'$(ArchiveTests)' == 'true'" >
+    <PropertyGroup>
+      <RunScriptCommand>
+        $HARNESS_RUNNER wasm $XHARNESS_COMMAND  --app=. --engine=$(JSEngine) $(JSEngineArgs) --js-file=runtime.js --output-directory=$XHARNESS_OUT --  $(RunTestsJSArguments) --run $(AssemblyName).dll
+      </RunScriptCommand>
+    </PropertyGroup>
+
+    <MSBuild Projects ="$(RepoRoot)eng/testing/tests.targets"
```

Comment:
Why not import the `tests.targets` file and just set your target to depend on `GenerateRunScript`? Would that work?

Reply:
If the "tests.targets" file is imported into the project, there will be such error during build: 
`error MSB4011: "/runtime/src/mono/wasm/build/WasmApp.targets" cannot be imported again. It was already imported at "/runtime/src/mono/netcore/sample/wasm/console/WasmSample.csproj (73,3)". This is most likely a build authoring error. This subsequent import will be ignored.`

Reply:
I see... what you could do is remove the `WasmApp.targets` import in `WasmSample.csproj` and then that issue would be gone. 

https://github.com/dotnet/runtime/blob/e81753c6542d1a45d6bab0020837053c5e3f05cc/src/mono/netcore/sample/wasm/console/WasmSample.csproj#L87